### ISSUE TITLE
cebe/markdown ~1.2.0 compatibility

### ISSUE THREAD

**Issue by ricpelo** [cebe/markdown-latex#14]
I'm unable to install `cebe/markdown-latex` with `cebe/markdown` ~1.2.1, because `cebe/markdown-latex` depends on `cebe/markdown` ~1.0.0:

https://github.com/cebe/markdown-latex/blob/42eb55c6f5a8dd68a1c029755eccd73e117aaa9b/composer.json#L23

Is it safe to bump `cebe/markdown` dependency to ~1.2.0?

**Comment by cebe** [comment]
no, there have been changes in internal structure that break the markdown-latex parser. It needs to be adjusted to work with newer versions.

**Comment by ricpelo** [comment]
I'll wait, then. Thanks!

**Comment by hooman-mirghasemi** [comment]
@cebe 
Is there a specific schedule to solve this problem?
I need yii2-api-doc but I can't install it because of this problem?

**Comment by cebe** [comment]
I have no schedule for this, sorry. What exactly is blocking you from using yii2-apidoc? composer should be able to install a set of package versions that work together.

**Comment by hooman-mirghasemi** [comment]
look at https://github.com/yiisoft/yii2-apidoc/issues/183
so i run composer update -vvv to fix this problem.

**Comment by cebe** [comment]
yes, that is the correct solution.
